db/migrate/20200429190910_add_partner_information_to_intake.rb

```diff
@@ -0,0 +1,15 @@
+class AddPartnerInformationToIntake < ActiveRecord::Migration[6.0]
+  def change
+    add_column :intakes, :vita_partner_name, :string
+
+    reversible do |dir|
```

Comment:
interesting - is this a backfill combined with a migration? never seen anything like this